src/coreclr/tools/aot/crossgen2/Properties/Resources.resx

```diff
@@ -390,4 +390,7 @@
   <data name="CpuFamilies" xml:space="preserve">
     <value>The following CPU names are predefined groups of instruction sets and can be used in --instruction-set too:</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="Crossgen2BannerText" xml:space="preserve">
+    <value>Microsoft (R) .NET Crossgen2 Compiler</value>
```

Comment:
We got rid of the `Microsoft (R)` banner from .NET SDK tools. Could you please remove it from crossgen2 and ilc as well?

Reply:
Also removed from il{d}asm.